test-project/Assets/Generated/Source/improbable/testschema/TypeC.cs

```diff
@@ -12,27 +12,18 @@ namespace Improbable.TestSchema
     [global::System.Serializable]
     public struct TypeC
     {
-        public global::Improbable.TestSchema.TypeB? BOption;
         public global::System.Collections.Generic.List<global::Improbable.TestSchema.TypeB> BList;
         public global::System.Collections.Generic.Dictionary<string,global::Improbable.TestSchema.TypeB> BMap;
     
-        public TypeC(global::Improbable.TestSchema.TypeB? bOption, global::System.Collections.Generic.List<global::Improbable.TestSchema.TypeB> bList, global::System.Collections.Generic.Dictionary<string,global::Improbable.TestSchema.TypeB> bMap)
```

Comment:
Looks like I forgot to regen the schema the second time for https://github.com/spatialos/gdk-for-unity/pull/1131